.github/workflows/cpp-packaging.yml

```diff
@@ -35,12 +39,31 @@ env:
   xcodeVersion: "12"
 
 jobs:
+  prepare_matrix:
+    runs-on: ubuntu-latest
+    env:
+      MATRIX_BUILD_TOOLS_OS: 'ubuntu-latest,macos-latest'
+      EXPANDED_MATRIX_BUILD_TOOLS_OS: 'ubuntu-latest,macos-latest'
+    outputs:
+      matrix_build_tools_os: ${{ steps.export-result.outputs.matrix_build_tools_os }}
+    steps:
+      - name: Use expanded matrix
+        if: github.event.inputs.expanded_matrix == '1'
```

Comment:
Also use expanded matrix if this is a build triggered by cron?

Reply:
Done